nvflare/private/fed/server/server_engine.py

```diff
@@ -707,14 +707,14 @@ def reset_errors(self, job_id) -> str:
     def _send_admin_requests(self, requests, timeout_secs=10) -> List[ClientReply]:
         return self.server.admin_server.send_requests(requests, timeout_secs=timeout_secs)
 
-    def check_client_resources(self, job_id: str, resource_reqs) -> Dict[str, Tuple[bool, str]]:
+    def check_client_resources(self, job: Job, resource_reqs, fl_ctx: FLContext) -> Dict[str, Tuple[bool, str]]:
         requests = {}
         for site_name, resource_requirements in resource_reqs.items():
             # assume server resource is unlimited
             if site_name == "server":
                 continue
-            request = Message(topic=TrainingTopic.CHECK_RESOURCE, body=resource_requirements)
-            request.set_header(RequestHeader.JOB_ID, job_id)
+            request = self._make_message_for_check_resource(job, resource_requirements, fl_ctx)
+
```

Comment:
How do we send public props in the fl_ctx to the client side thru admin server?

Reply:
admin server sending messages to each client will include the fl_ctx public data in the peer_ctx.